alacritty_terminal/src/window.rs

```diff
@@ -409,6 +423,51 @@ impl Window {
     }
 }
 
+#[cfg(not(any(target_os = "macos", windows)))]
+fn x_embed_window(window: &GlutinWindow, parent_window_id: u64) {
+    let (xlib_display, xlib_window) = match (window.get_xlib_display(), window.get_xlib_window()) {
+        (Some(display), Some(window)) => (display, window),
+        _ => return,
+    };
+
+    let xlib = Xlib::open().expect("get xlib");
+
+    unsafe {
+        let xembed = "_XEMBED".as_ptr();
+        let atom = (xlib.XInternAtom)(xlib_display as *mut _, xembed as *const _, 0);
+        (xlib.XChangeProperty)(
+            xlib_display as _,
+            xlib_window as _,
+            atom,
+            atom,
+            32,
+            PropModeReplace,
+            [0, 1].as_ptr(),
+            2,
+        );
+        
+        // set a handler before verifying target window existence
```

Comment:
Oh, just noticed. We're starting our `//` comments with uppercase later. So should be `Set` etc.

Reply:
Oh yeah, noticed that too but forget to mention it. Thanks for pointing it out.

Reply:
Sorted :)